CHANGELOG.md

```diff
@@ -9,8 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased][Unreleased_log]
 --------------
-### Deprecated
+### Added
+- Add the deep-copy out parameter support as an experimental, SGX-only feature. To use the feature, passing `--experimental` when invoking oeedger8r. Refer to the [design document](docs/DesignDocs/DeepCopyOutParameters.md) for more detail.
```

Comment:
```suggestion
- Add the deep-copy out parameter support as an experimental, SGX-only feature. To use the feature, pass `--experimental` when invoking oeedger8r. Refer to the [design document](docs/DesignDocs/DeepCopyOutParameters.md) for more detail.
```

Reply:
What happens on non-SGX?  Does compilation fail with an appropriately helpful message?   Or does it fail gracefully at runtime?  Or does it just crash unexpectedly?

Reply:
The feature is currently guarded by the `--experimental` flag only. The tool generates the following message if the flag is not passed:
```
`out' for user defined type `MyStruct\\*' requires the --experimental option
```

The oeedger tool does not generate TEE-specific message as it currently does not detect the target platform. However, even if developers try to enable the feature in the non-SGX case, neither compilation nor runtime fails. Just that the deep-copy will not work as expected. We could improve the user experience when we make the feature as non-experimental.